recipes/llvm-core/all/CMakeLists.txt

```diff
@@ -4,4 +4,20 @@ project(conanllvm)
 include(${CMAKE_CURRENT_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup(KEEP_RPATHS)
 
+if(LLVM_ENABLE_ZLIB)
+  find_package(ZLIB QUIET)
+
+  # The FindZLIB find module provided by CMake behaves differently than the FindZLIB.cmake script generated by CONAN.
+  # So the following two lines fill in the needed properties.
+  list(GET ZLIB_LIBRARIES 0 ZLIB_LIBRARY)
+  set_property(TARGET ZLIB::ZLIB PROPERTY LOCATION "${ZLIB_LIBRARY}")
```

Comment:
Can we do this `FindZLIB.cmake` generated by Conan?
There may exist products that assume `FindZLIB.cmake` provided by CMake.

Reply:
We would need to update the zlib recipe with a "test module package" and make sure the variables are correctly defined